lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -403,7 +403,8 @@ def get_javascript(cls, stream=None):
         for filetype, ext in sorted(FigureCanvasWebAggCore.
                                     get_supported_filetypes_grouped().
                                     items()):
-            extensions.append(ext[0])
+            if not ext[0] == 'pgf':  # pgf does not support BytesIO
```

Comment:
At some point I added support for witing to targets that are `matplotlib.cbook.is_writable_file_like`. Yet writing `pgf` instead of `pdf` or `png` to a stream raises a UserWarning since there is no way of embedding rastered images within PGF.